torch/package/package_exporter.py

```diff
@@ -430,11 +432,32 @@ def add_dependency(self, module_name: str, dependencies=True):
             )
             return
 
+
+        # SELECTIVE INTERN START
+        if module_name == "torch":
```

Comment:
the place where I stopped was: I'd like to make selective intern generic (not hardcoded to be torch), so that's easier to unit test.